Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsSpeechInputProvider.cs

```diff
@@ -157,19 +157,24 @@ public override void Disable()
             {
                 StopRecognition();
                 keywordRecognizer.OnPhraseRecognized -= KeywordRecognizer_OnPhraseRecognized;
+
+                keywordRecognizer.Dispose();
             }
+
+            keywordRecognizer = null;
         }
 
-#if UNITY_EDITOR
         /// <inheritdoc />
         public override void Destroy()
         {
+            Disable();
```

Comment:
`Disable` should only really be called from the service's manager. We shouldn't need to call it here.
Is a manager (maybe `MixedRealityToolkit.cs`?) trying to destroy services that it hasn't disabled yet?

Reply:
Yes, that was a problem in `MixedRealityToolkit.cs`. Added Disable call there. Thanks for the comment!